db/__init__.py

```diff
@@ -78,3 +78,18 @@ class TransactionStatusMemo(Base):
     id = Column(Integer, primary_key=True)
     memo_body = Column(String, nullable=False)
     tx_signatures_id = Column(Integer, ForeignKey(f'{SCHEMA}.tx_signatures.id'), nullable=False)
+
+
+class AmmLiquidity(Base):
+    __tablename__ = 'amm_liquidity'
```

Comment:
```    __table_args__ = (
        PrimaryKeyConstraint("dex", "pair", "market_address", "timestamp"),
        {"schema": SCHEMA},
    )``` not sure, but this might be missing

Reply:
agree